### ISSUE TITLE
[ENHANCEMENT] Select Bible reference when pressing ctrl + l, even when the focus is on the box.

### ISSUE THREAD

**Issue by entodoays** [bibletime/bibletime#353]
When navigating from one bible text to another, many times I use the ctrl + l shortcut and I find it very handy: just press the shortcut and type away. However, when the reference box is already in focus, pressing ctrl + l does nothing and if I start typing it add text to the existing reference. It is no big deal, but I think that making ctrl + l automatically highlight all the text in the box makes this feature better.

**Comment by jaakristioja** [comment]
I agree. This would be more familiar for users of graphical web browsers.
